ajax/libs/mdui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "mdui",
+  "filename": "mdui.min.js",
+  "description": "a CSS Framework based on material design",
+  "keywords": [
+    "material design",
+    "responsive",
+    "framework",
+    "front-end",
+    "mobile-first",
+    "web"
+  ],
+  "author": {
+    "name": "zdhxiong",
+    "email": "zdhxiong@gmail.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zdhxiong/mdui.git"
+  },
+  "homepage": "http://mdui.org",
+  "npmName": "mdui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
@zdhxiong We should exclude `.txt` and `.ijmap` files. Let's use `"**/*.!(txt|ijmap)"`. Thanks.